### ISSUE TITLE
AttributeError in QueryMyChem

### ISSUE THREAD

**Issue by saramsey** [RTXteam/RTX#255]
Running BuildMasterKG.py on rtxsteve.saramsey.org, I see the following error:

```
Traceback (most recent call last):
  File "BuildMasterKG.py", line 137, in <module>
    running_time = timeit.timeit(lambda: run_function(), number=1)
  File "/usr/lib/python3.5/timeit.py", line 213, in timeit
    return Timer(stmt, setup, timer, globals).timeit(number)
  File "/usr/lib/python3.5/timeit.py", line 178, in timeit
    timing = self.inner(it, self.timer)
  File "<timeit-src>", line 6, in inner
  File "BuildMasterKG.py", line 137, in <lambda>
    running_time = timeit.timeit(lambda: run_function(), number=1)
  File "BuildMasterKG.py", line 94, in make_master_kg
    bne.expand_all_nodes()
  File "/mnt/data/orangeboard/RTX/code/reasoningtool/kg-construction/BioNetExpander.py", line 609, in expand_all_nodes
    self.expand_node(node)
  File "/mnt/data/orangeboard/RTX/code/reasoningtool/kg-construction/BioNetExpander.py", line 597, in expand_node
    expand_method(self)
  File "/mnt/data/orangeboard/RTX/code/reasoningtool/kg-construction/BioNetExpander.py", line 207, in expand_chemical_substance
    res_dict = DrugMapper.map_drug_to_ontology(node.name)
  File "/mnt/data/orangeboard/RTX/code/reasoningtool/kg-construction/DrugMapper.py", line 114, in map_drug_to_ontology
    drug_use = QueryMyChem.get_drug_use(chembl_id)
  File "/mnt/data/orangeboard/RTX/code/reasoningtool/kg-construction/QueryMyChem.py", line 232, in get_drug_use
    if 'contraindication' in drug_uses.keys():
AttributeError: 'list' object has no attribute 'keys'
```

**Comment by DeqingQu** [comment]
@saramsey 
As I mentioned in Issue #249, the 'AttributeError' bug is caused by the dirty cache. Because the format of response data of MyChem has been changed by the MyChem backend, the previous cache is not valid anymore.
But I am not very sure where the cache for MyChem is located. It depends on the execution sequence of the previous commands. It can be in 'SynonymCache.sqlite', 'QueryMyChem.sqlite' or 'orangeboard.sqlite'.
In the 'rtxsteve' docker container, I deleted all three cache files and the bug didn't happen when I ran the BuildMasterKG.py. The BuildMasterKG.py is still running and I don't know how long it will take.
